### PR TITLE
update security considerations

### LINKED ISSUE #199: Re-work Security considerations
- Only cryptographically protected variants are considered --> adjust security considerations accordingly if necessary

### PR DESCRIPTION
**Pull request by c2bo**
Closes #199

### CODE REVIEW

File: draft-ietf-oauth-status-list.md [hunk 1]
```diff
@@ -818,6 +818,8 @@ Resulting in the byte array and compressed/base64url-encoded Status List:
 
 # Security Considerations {#Security}
 
+The Status List as defined in this specification only exists in cryptographically secured containers which allows checking the integrity and origin without relying on other aspects like transport security (e.g., the web PKI).
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:821]: this is a little confusing to me, are we talking about the Status List object, that is only communciated in Status List Tokens or about referenced Tokens?

Comment (c2bo) [reply draft-ietf-oauth-status-list.md:821]: status list object currently (after the change) only exists in secured containers

Comment (paulbastian) [reply draft-ietf-oauth-status-list.md:821]: should we add reference to the section?

**Review by paulbastian — APPROVED** [review]

File: draft-ietf-oauth-status-list.md (hunk 1 above)
Comment (c2bo) [reply draft-ietf-oauth-status-list.md:821]:
```suggestion
The Status List as defined in [](#status-list) only exists in cryptographically secured containers which allows checking the integrity and origin without relying on other aspects like transport security (e.g., the web PKI).
```

**Review by tplooker — APPROVED** [review]

**Pull request merged**